test/unitTests/core/test-issuers.js

```diff
@@ -39,6 +39,26 @@ test('split good amounts', t => {
   }
 });
 
+test('split good w/different amounts', t => {
```

Comment:
Would it be possible to take this test out of this PR since we have a SwingSet based test for the bug on master now?

Reply:
sure.